pkg/smart_wallet/README.md

```diff
@@ -0,0 +1,15 @@
+# SmartWallet GoLang library
+
+## Overview
+
+SmartWallet is a golang library that provides helper functions to user operations.
+
+## Features
+
+This package introduces the following types and helper functionality:
+
+- `Config` - A struct that holds the configuration for the SmartWallet.
+- `Type` - an enum that represents the type of the wallet. Currently only ZeroDev `Kernel`, `Biconomy` and eth-infinitism `SimpleWallet` are supported.
```

Comment:
Let's add markdown links to smart wallets' repositories

Reply:
And maybe list the exact versions for each wallet type that are confirmed to work fine?

Reply:
747c10e4be8f925e323c6812a63179a4291a6819